ext/php_driver.h

```diff
@@ -460,6 +460,9 @@ php5to7_string_compare(php5to7_string s1, php5to7_string s2)
 
 extern zend_module_entry php_driver_module_entry;
 
+#define cassandra_module_ptr &php_driver_module_entry
+#define phpext_cassandra_ptr cassandra_module_ptr
```

Comment:
Could you please update `#define cassandra_module_ptr &php_driver_module_entry` => **`#define phpext_cassandra_ptr &php_driver_module_entry`** and then remove **`#define phpext_cassandra_ptr cassandra_module_ptr`**.